core/hosting/README.md

```diff
@@ -1,27 +1,13 @@
-.NET Core Hosting Sample
-==================================
+Sample .NET Core Hosts
+======================
 
-This sample demonstrates a very simple .NET Core host. It is part of the [.NET Core hosting tutorial](https://docs.microsoft.com/dotnet/core/tutorials/netcore-hosting). Please see that topic for a more detailed explanation of this sample.
+This folder contains sample code demonstrating how to host managed .NET Core code in a native process. These hosts bypass the usual `dotnet` host and launch managed code directly.
 
-About .NET Core Hosts
----------------------
+There are two samples demonstrating two different hosting interfaces for .NET Core.
 
-.NET Core applications are always run by a host. In most cases, the default dotnet.exe host is used.
+1. The [HostWithCoreClrHost](HostWithCoreClrhost) folder demonstrates how to host the .NET Core runtime using the newer CoreCLRHost.h API. This API is the preferred method of hosting .NET Core and is a bit simpler to use than mscoree.h. This host demonstrates calling from native code into a static managed method and supplying a function pointer for the managed code to use to call back into the host.
+1. The [HostWithMscoree](HostWithMscoree) folder contains a sample host using the `ICLRRuntimeHost2` interface from msocree.h to host .NET Core. This is the interface that was originally used to host .NET Core and it still has a few more APIs than the CoreClrHost alternative (including unsupported features like creating multiple AppDomains). This host demonstrates launching a managed executable from a native host application.
```

Comment:
Core and it still has --> Core. It still has

Reply:
Fixed